File: practice/recursive.py
def fibo(n):
  if n < 2:
    return 1
  else:
    return fibo(n - 1) + fibo(n - 2)
    

print(fibo(2))
print(fibo(3))
print(fibo(4))

def factorial(n):
  if n < 2:
    return 1
  else:
    return n * factorial(n-1)
    
print(factorial(2))
print(factorial(3))
print(factorial(4))

File: practice/recursive.js
function fibo(int) {
	if (int < 2) {
		return 1;
	} else {
		return fibo(int - 1) + fibo(int - 2);
	}
}

console.log(fibo(3))
console.log(fibo(4))
console.log(fibo(5))

function factorial(int) {
	if (int < 2) {
		return 1;
	} else {
		return int * factorial(int - 1);
	}
}

console.log(factorial(3))
console.log(factorial(4))
console.log(factorial(5))
